src/app/api/application/uniform/route.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  createApplication,
+  updateApplication,
+  checkExistingReference,
+} from "@/app/actions/sanityClient";
+import { verifyApiKey } from "../../../lib/apiKey";
+import { NextRequest, NextResponse } from "next/server";
+import {
+  validateUniformData,
+  applicationNumberValidation,
+} from "@/app/actions/uniformValidator";
+
+interface ApplicationError {
+  application: any;
+  error: string;
+}
+
+interface ApplicationResult {
+  success: string[];
+  errors: ApplicationError[];
+}
+
+/**
+ * @swagger
+ * /api/application/uniform:
+ *   put:
+ *     summary: Insert new planning application
```

Comment:
Could this change to something like 'Insert a new planning application or update one if it already exists' ? 😊